valhalla/configuration.h

```diff
@@ -0,0 +1,49 @@
+#ifndef VALHALLA_CONFIGURATION_H_
+#define VALHALLA_CONFIGURATION_H_
+
+#include <stdexcept>
+#include <string>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include "baldr/rapidjson_utils.h"
+#include "filesystem.h"
```

Comment:
since we are on c++17 now you can use std::filesystem instead. we should try to get away from using the our port over time.